include/deal.II/base/mpi_remote_point_evaluation.h

```diff
@@ -156,6 +156,18 @@ namespace Utilities
       bool
       is_map_unique() const;
 
+      /**
+       * Return if all points could be associated to any process.
+       */
+      bool
+      all_points_found() const;
+
+      /**
+       * Return for each point passed to reinit() if it has been found or not.
+       */
+      std::vector<bool>
+      valid_point_masks() const;
```

Comment:
The information that "point i has been found == (vector[i] = true)" is somewhat implicit here (i.e. what is a "valid" point?). I wonder whether one could find a better name for the function. I make some suggestions: `individual_points_found()`, `vector_points_found()`, `which_points_found()`? (my goal is that the name is is intuitive to understand in combination with `all_points_found()`).